Makefile

```diff
@@ -51,6 +51,49 @@ dev: ## dev
 		$(eval var_file=$(env))
 		$(eval backend_config=-backend-config="key=$(env)/app.tfstate")
 
+
+.PHONY: development_aks
+development_aks:
```

Comment:
We are focusing on review, this can be removed for now

Reply:
This has been removed